build-configs/integreat/index.js

```diff
@@ -14,6 +14,7 @@ const APPLE_APP_ID = '1072353915'
 
 const commonIntegreatBuildConfig: CommonBuildConfigType = {
   appName: 'Integreat',
+  appDescription: 'Daily Guide for Refugees. Digital. Multilingual. Free.',
```

Comment:
Maybe we should use german here?

Reply:
What would be the correct german slogan? 
Should i use this: Integreat – die lokale und mehrsprachige Integrations-Plattform für Zugewanderte (its from the integreat homepage)